include/tables/bombers_notebook/event_table.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * Bombers Notebook Event Table
+ *
+ * DEFINE_EVENT should be used for events in the notebook
+ *    - Argument 0: Enum value for the event (without the pfefix)
+ *    - Argument 1: Icon to use (see BombersNotebookEventIcon)
+ *    - Argument 2: WeekEventFlag that updates the icons color
+ *    - Argument 3: Message id for the description of this event
+ *    - Argument 4: Message id for the completion of this event
+ *    - Argument 5: WeekEventFlag for the completion of this event
+ */
+/* 0x00 */ DEFINE_EVENT(RECEIVED_ROOM_KEY, BOMBERS_NOTEBOOK_EVENT_ICON_EXCLAMATION_POINT, WEEKEVENTREG_75_10, 0x2198, 0x2152, WEEKEVENTREG_BOMBERS_NOTEBOOK_EVENT_RECEIVED_ROOM_KEY)
```

Comment:
Could these weekEventFlags/arg 5 be named here based on the event? Or would you rather do that separately? (i.e. `WEEKEVENTREG_75_10` and others)

Reply:
I'm actually not quite sure what to name them. They are meant to be for the event, but on a temporary basis per cycle. That way it changes color when you have done it for that cycle.
Perhaps just use `BOMBERS_NOTEBOOK_EVENT_COLOR_`, but not 100% sure on that. If we can figure out a name I wouldn't mind doing it though.

Reply:
Decided to go with actually naming them the same as the permanent Event flags, but without the BOMBERS_NOTEBOOK_EVENT prefix protion, since these are more general and used elsewhere.